plugins/log_request_plugin/src/main.rs

```diff
@@ -71,13 +68,3 @@ async fn main() -> Result<(), anyhow::Error> {
 
     Ok(())
 }
-
-/// does a one second pause.
-///
-/// NOTE: this is a quick-fix to enable step 4. in main to be executed.
-/// if `do_one_second_pause()` is not called, step 4. in main
-/// does not get triggered.
-// see #846
-async fn do_one_second_pause() {
-    sleep(Duration::from_secs(1)).await;
-}
```

Comment:
This is already done in a separate PR. Isn't it?

Reply:
> Do you have a clue of why the tests are failing?

This is a mystery for me. The panics are due to not receiving all the expected messages, but I can't understand why is this happening (they seem to break only in the CI).
My last couple of commits was to understand better what is going on. They started to break after rebase.